src/jsonUtils/computeNode.js

```diff
@@ -0,0 +1,445 @@
+      yogaNode.setBorder(yoga.EDGE_BOTTOM, style.borderBottom);
+    }
+    if (!isNullOrUndefined(style.borderLeft)) {
+      yogaNode.setBorder(yoga.EDGE_LEFT, style.borderLeft);
+    }
+    if (!isNullOrUndefined(style.borderRight)) {
+      yogaNode.setBorder(yoga.EDGE_RIGHT, style.borderRight);
+    }
+    if (!isNullOrUndefined(style.borderVertical)) {
+      yogaNode.setBorder(yoga.EDGE_VERTICAL, style.borderVertical);
+    }
+    if (!isNullOrUndefined(style.borderHorizontal)) {
+      yogaNode.setBorder(yoga.EDGE_HORIZONTAL, style.borderHorizontal);
+    }
+    if (!isNullOrUndefined(style.border)) {
+      yogaNode.setBorder(yoga.EDGE_ALL, style.border);
+    }
+
+    // Flex
+    if (!isNullOrUndefined(style.flex)) {
+      yogaNode.setFlex(style.flex);
+    }
+    if (!isNullOrUndefined(style.flexGrow)) {
+      yogaNode.setFlexGrow(style.flexGrow);
+    }
+    if (!isNullOrUndefined(style.flexShrink)) {
+      yogaNode.setFlexShrink(style.flexShrink);
+    }
+    if (!isNullOrUndefined(style.flexBasis)) {
+      yogaNode.setFlexBasis(style.flexBasis);
+    }
+
+    // Position
+    if (style.position === 'absolute') {
+      yogaNode.setPositionType(yoga.POSITION_TYPE_ABSOLUTE);
+    }
+    if (style.position === 'relative') {
+      yogaNode.setPositionType(yoga.POSITION_TYPE_RELATIVE);
+    }
+
+    if (!isNullOrUndefined(style.top)) {
+      if (isPercent(style.top)) {
+        yogaNode.setPositionPercent(yoga.EDGE_TOP, style.top);
+      } else {
+        yogaNode.setPosition(yoga.EDGE_TOP, style.top);
+      }
+    }
+    if (!isNullOrUndefined(style.left)) {
+      if (isPercent(style.left)) {
+        yogaNode.setPositionPercent(yoga.EDGE_LEFT, style.left);
+      } else {
+        yogaNode.setPosition(yoga.EDGE_LEFT, style.left);
+      }
+    }
+    if (!isNullOrUndefined(style.right)) {
+      if (isPercent(style.right)) {
+        yogaNode.setPositionPercent(yoga.EDGE_RIGHT, style.right);
+      } else {
+        yogaNode.setPosition(yoga.EDGE_RIGHT, style.right);
+      }
+    }
+    if (!isNullOrUndefined(style.bottom)) {
+      if (isPercent(style.bottom)) {
+        yogaNode.setPositionPercent(yoga.EDGE_BOTTOM, style.bottom);
+      } else {
+        yogaNode.setPosition(yoga.EDGE_BOTTOM, style.bottom);
+      }
+    }
+
+    // Display
+    if (style.display) {
+      if (style.display === 'flex') {
+        yogaNode.setDisplay(yoga.DISPLAY_FLEX);
+      }
+      if (style.display === 'none') {
+        yogaNode.setDisplay(yoga.DISPLAY_NONE);
+      }
+    }
+
+    // Overflow
+    if (style.overflow) {
+      if (style.overflow === 'visible') {
+        yogaNode.setDisplay(yoga.OVERFLOW_VISIBLE);
+      }
+      if (style.overflow === 'scroll') {
+        yogaNode.setDisplay(yoga.OVERFLOW_SCROLL);
+      }
+      if (style.overflow === 'hidden') {
+        yogaNode.setDisplay(yoga.OVERFLOW_HIDDEN);
+      }
+    }
+
+    // Flex direction
+    if (style.flexDirection) {
+      if (style.flexDirection === 'row') {
```

Comment:
Would it be worth extracting these into an object of `flexFirectionModes = { row: yoga.FLEX_DIRECTION_ROW ... }`?

Reply:
See my previous comment.